docs/cli/get.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "wash get"
+draft: false
+sidebar_position: 11
+description: "wash get command reference"
+--- 
+
+Wash get helps you in retrieving information about your lattice. It can help you retrieve all the links in the lattice, hosts, their inventories and claims. Following are the subcommands available under `wash get`:
+
+- `links`
+- `hosts`
+- `inventory`
+- `claims`
+
+### `links`
+
+This subcommand will retrieve all the known links in the lattice. It will give you information about them such as the link name, the associated actor and provider ID and its contract ID.
+
+#### Usage
+```
+wash get links
+```
+
+### `hosts`
+
+This subcommand retrieves all the hosts currently running in the lattice. You will be able to view each hosts host ID and its up time.
```

Comment:
Should be "uptime" I believe. If spell check is telling you no, then don't worry about it